Taskfile.yml

```diff
@@ -3,27 +3,79 @@
 version: '3'
 
 vars:
-  # 'VERSION_DYNAMIC' needs to run in this scope before 'task build' begins
+  OPSLEVEL_GO_PKG: "github.com/opslevel/opslevel-go/v2024"
+  # 'VERSION_DYNAMIC' needs to run in this scope before 'task terraform-build' begins
   VERSION_DYNAMIC:
     sh: echo "$(date +%s).0.0"
   WORKSPACE_DIR: "workspace"
 
-includes:
-  go:
-    internal: true
-    taskfile: "./opslevel/Taskfile.yml"
-
 tasks:
+  ci:
+    desc: Workflow to run in CI
+    deps:
+      - brew-install-jq
+      - install-gofumpt
+      - install-golangci-lint
+      - install-nilaway
+    cmds:
+      - task: workspace
+      - task: has-latest-opslevel-go
+      - task: lint
```

Comment:
Small but worth pointing out. Let's move `task: lint` before `task: workspace`. (I think this update was made to `kubectl-opslevel` since that was taking a long time)

Putting `lint` first will ensure we're only lint checking the code in this repo, and not the submodule repos also. Plus it's a little bit faster.

Reply:
done

Reply:
after moving it before workspace CI failed because of unresolved references

https://github.com/OpsLevel/terraform-provider-opslevel/actions/runs/7762742933/job/21173702092